### ISSUE TITLE
Cant install ethersjs

### ISSUE THREAD

**Issue by eugenioclrc** [oven-sh/bun#438]
I have try to install ethers but its not working;

```
bun add ethers
```
Ends up in a frozen state.

**Comment by Pruxis** [comment]
It get's stuck on `@babel/plugin-transform-unicode-escapes [1728/1731]`

**Comment by Pruxis** [comment]
@eugenioclrc As a workaround install the babel plugin separately and then your installation should work.
However, I'm unsure if this will have implications on any of the functionality, although I would doubt so.

```
bun add @babel/plugin-transform-unicode-escapes
bun add ethers
```

**Comment by FinnRG** [comment]
@eugenioclrc  I'm unable to reproduce this issue. Are you using the latest version (v0.1.2)? If not, please upgrade using ```bun upgrade```.

**Comment by eugenioclrc** [comment]
i have just upgrade it, how ever, im still having the same issue;
![Captura de pantalla de 2022-07-08 11-52-21](https://user-images.githubusercontent.com/332426/178017022-91053bef-b1a1-4853-a8fa-92898ee1d14e.png)

**Comment by FinnRG** [comment]
What is your OS? And please create a verbose log (```bun add ethers --verbose```).

**Comment by eugenioclrc** [comment]
Ubuntu

**Comment by Pruxis** [comment]
@FinnRG I can also no longer reproduce after upgrading (was on 0.1.1) at time of testing.

**Comment by FinnRG** [comment]
> i have just upgrade it, how ever, im still having the same issue; ![Captura de pantalla de 2022-07-08 11-52-21](https://user-images.githubusercontent.com/332426/178017022-91053bef-b1a1-4853-a8fa-92898ee1d14e.png)

@eugenioclrc  Could you please give us your kernel version? (```uname -r```). Also please provide a verbose log (```bun add ethers --verbose > log.txt```)

**Comment by Karl-EdwardJeanMehu** [comment]
Bun does indeed hang when installing ethersJS. I can confirm @Pruxis 's solution works.

**Comment by ttdatt** [comment]
I had same problem with `firebase-admin`, tried to install it, but end up with install 5 dependencies to make it work. 

Platform: MacOS 12.4 Apple M1
[logs.txt](https://github.com/oven-sh/bun/files/9079393/logs.txt)

This is my package.json.
````{
  "name": "node-fcm",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "@firebase/auth-interop-types": "^0.1.6",
    "@panva/asn1.js": "^1.0.0",
    "@protobufjs/codegen": "^2.0.4",
    "ansi-regex": "^6.0.1",
    "color-convert": "^2.0.1",
    "firebase-admin": "^11.0.0"
  }
}

**Comment by KKatowice** [comment]
For those who got this problem while installing ethers 
`error: Unsupported dependency type git for "git+https://github.com/ethereumjs/ethereumjs-abi.git"`
![image](https://user-images.githubusercontent.com/98989441/178467668-0ee4df70-d445-45e1-951d-1feb71591f1b.png)
On discord the answer was " git dependencies aren't supported yet " .
if anyone have any workaround to install ethers would be nice, ty.

**Comment by malonehedges** [comment]
i've been able to install ethers. not sure when it changed for me but i wasn't able to and have been able to recently. now i am just seeing other http issues using a RPC provider with ethers

**Comment by Jarred-Sumner** [comment]
The original issue was a missing error message for `git` dependencies. That issue still exists, but is tracked separately 

The HTTP client issues should be fixed as of Bun v0.1.12
